docs/guides/server/bootstrap-admin-recovery.adoc

```diff
@@ -0,0 +1,90 @@
+<#import "/templates/guide.adoc" as tmpl>
+<#import "/templates/kc.adoc" as kc>
+<#import "/templates/links.adoc" as links>
+
+<@tmpl.guide
+title="Admin bootstrap and recovery"
+summary="Learn how to bootstrap and recover admin account." >
+
+== A solution for a disaster scenario
```

Comment:
Bootstrapping is not just for disaster recovery scenarios.

Reply:
Adressed.